test/helpers/points-helper.js

```diff
@@ -0,0 +1,19 @@
+// TODO: Finish filling in these methods
```

Comment:
Are these TODOs for another story? Just highlighting in case they aren't. Otherwise, me gusta 👍